src/components/Counter/index.tsx

```diff
@@ -16,8 +16,22 @@ const styles = {
   `,
 }
 
-const Counter = ({ end, onEnd }) => {
-  const start = useRef(0)
+type CounterProps = {
+  /**
+   * Value to which counter will go. Can be negative or positive number.
+   */
+  end: number
+  /**
+   * Triggered function when counter animation is finished.
+   */
+  onEnd?: () => unknown
```

Comment:
```suggestion
  onEnd?: () => void
```

We won't care about the return no ?